### ISSUE TITLE
[TASK] Fix Lighthouse failure - Hub and spoke

### ISSUE THREAD

**Issue by rmccar** [ONSdigital/design-team#224]
Alessio addressed Lighthouse failure in the ticket https://github.com/ONSdigital/design-system/issues/3350.

As an outcome of the above ticket, identified a need to address the warnings for all below components. It relates to accessibility.

List of components

1 result(s) for http://localhost:39203/patterns/hub-and-spoke/example-spoke-summary.html
1 result(s) for http://localhost:39203/patterns/hub-and-spoke/example-hub.html
1 result(s) for http://localhost:39203/patterns/hub-and-spoke/example-hub-complete.html

**Action**
Use yarn start and then move onto the above components in order. Use the lighthouse test from the google chrome dev tools. Run the test on the component and only look at the accessibility score. Fix the issues highlighted under the accessibility section. Run the test again and confirm that they are now at 100. Move on to the next component.
